src/app/components_ngrx/iteration-list-entry/iteration-list-entry.component.html

```diff
@@ -14,8 +14,10 @@
     <div class="truncate padding-right-15"
       (click)="setGuidedTypeWI(collection)">
       <a [routerLink]="[]"
-        [queryParams]="showTree ?
-          { q: constructURL(iteration.id), showTree: showTree } :
+        [queryParams]="showTree && showCompleted ?
```

Comment:
This has become a bit complicated to understand. Let's create a function for this.